packages/js/api-core-tests/changelog/update-jest

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: update
+
+Update `jest` to `^27` from `^25`.
```

Comment:
I don't think this changelog entry is needed as the update was already made in https://github.com/woocommerce/woocommerce/pull/34345/ and the changelog updated as part of that
https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/api-core-tests/CHANGELOG.md

Reply:
Isn't that what JetPack Changelogger flow is for, to avoid such conflicts? Also, this PR and file were made before https://github.com/woocommerce/woocommerce/pull/34345.

Should I remove this file? Would PR check be satisfied, given I change something in `api-core-test` w/o providing a changelog note?

Reply:
> Should I remove this file? Would PR check be satisfied, given I change something in api-core-test w/o providing a changelog note?

I think so as looking further into it I don't believe `api-core-tests` uses the changelogger, it was excluded as it already had an existing workflow for updating the changelog.

Reply:
Thanks, for pointing that out and the explanations :) 
Removed in https://github.com/woocommerce/woocommerce/pull/34322/commits/2033bb24a7017b2b56f6752be828b09894271ce8